pkg/pool-stable/test/ComposableStablePoolProtocolFees.test.ts

```diff
@@ -598,6 +605,214 @@ describe('ComposableStablePoolProtocolFees', () => {
           }
         }
       });
+
+      describe('payProtocolFeesAfterJoinExit', () => {
```

Comment:
I like how you've set this up.

Reply:
Do you need any help with the skipped test cases?

Reply:
Thank Nico :) Just copied his structure.